ops/model.py

```diff
@@ -2275,6 +2275,9 @@ def local_list(source_path: Path) -> List[pebble.FileInfo]:
             try:
                 for info in Container._list_recursive(local_list, source_path):
                     dstpath = self._build_destpath(info.path, source_path, dest_dir)
+                    if info.type is pebble.FileType.DIRECTORY:
```

Comment:
It wasn't clear to me that enum values should be compared by identity (`is` vs `==`), but it looks like they should be -- TIL! For reference: https://docs.python.org/3/howto/enum.html#comparisons